eng/targets/ResolveReferences.targets

```diff
@@ -132,7 +132,9 @@
     This target resolves remaining Referene items to Packages, if possible. If not, they are left as Reference items fo the SDK to resolve.
     This executes on NuGet restore and during DesignTimeBuild. It should not run in the outer, cross-targeting build.
    -->
-  <Target Name="ResolveCustomReferences" BeforeTargets="CollectPackageReferences;ResolveAssemblyReferencesDesignTime;ResolveAssemblyReferences" Condition=" '$(TargetFramework)' != '' AND '$(EnableCustomReferenceResolution)' == 'true' ">
+  <Target Name="ResolveCustomReferences"
+      BeforeTargets="CheckForImplicitPackageReferenceOverrides;CollectPackageReferences;ResolvePackageAssets"
```

Comment:
@dougbu should this logic be merged forward from 3.1 to 5.0? It's causing a bunch of failures on CI.

Reply:
This change should be fine but none of the added block just below should be in 'master'.

Reply:
Deleting from line 219 to 265 then 😄